import "alice/rtlog/protos/rtlog.ev";

package NRTLogAgent;

message TFrameIndexKey {
    optional uint64 Key = 1;
    optional uint32 EventIndex = 2;
}

message TFrameInfo {
    optional bytes ReqId = 1;
    optional int64 ReqTimestamp = 2;
    repeated TFrameIndexKey IndexKeys = 3;
    optional NRTLogEvents.InstanceDescriptor Instance = 4;
    optional uint32 SegmentTimestamp = 5;
    optional uint32 Offset = 6;
    optional uint32 Size = 7;
}
